### DISCUSSION TITLE
Uploading to dockerhub

### DISCUSSION THREAD

**Discussion by KendaUrban** [FieldRobotEvent/Virtual_Field_Robot_Event#58] [Q&A]
When running `bash upload_my_container_publicly.sh 1` 
I got an error 
`upload_my_container_publicly.sh: line 6: =1: command not found`
`Uploading image for task `
`Please enter your hub.docker.com username here so I can tag your image nicely.
If you don't have one, break this and go to https://hub.docker.com to get one sorted.
`
`Username: MY_USERNAME`
`Error parsing reference: "b_task_" is not a valid repository/tag: invalid reference format
Tagged. Now prompting for login to hub.docker.com ...`
`Password: `
`Login Succeeded`
`Pushing now to hub.docker.com ...
invalid reference format`
`Done. Please let the organisers know to use the container called urbankenda/b_task_`

What am I doing wrong? Both images are present.

**Comment by ThijsRuigrok** [comment]
We recommend you to use the `dump_my_container.sh` script and upload the .tgz file using the link send to you earlier. 

https://github.com/FieldRobotEvent/Virtual_Field_Robot_Event/blob/master/competition_environment/dump_my_container.sh
